final-project/static/scripts.js

```diff
@@ -35,7 +71,7 @@ function editProduct() {
 
 
 // Updates product search and/or product specific changes.
-function update() {
+function updateDisplay(product_json) {
```

Comment:
productJson. Javascript is typically camelCased.

Reply:
Even for variables? I thought functions were camel cased, but variables were under_scored. 

If I make the updatedDisplay() function and the productJSON argument variable camel cased, won't it be confusing?

Reply:
everyhing except for classes in JS are camel cased